test_tf2/test/test_convert.py

```diff
@@ -34,6 +34,9 @@
 #* 
 #* Author: Eitan Marder-Eppstein
 #***********************************************************
+
+from __futute__ import print_function
```

Comment:
Looks like there is a typo: `__futute__` => `__future__`. (same below)
Nice catch for the python functions!